src/Function/Construct/IsEquivalence.agda

```diff
@@ -0,0 +1,63 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some functional properties are Equivalence Relations
+--   This file is meant to be imported qualified.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Function.Construct.IsEquivalence where
```

Comment:
The `Function.Construct` directory isn't quite the right location for this as it's not constructing new members of the function hierarchy. Instead I think the standard location for these proofs would be in the `Function.Properties.Inverse` and `Function.Properties.Equivalence` modules.

Reply:
Ok, moving them there.